MEV_spec_v0_2.md

```diff
@@ -20,10 +20,38 @@ We believe that without the adoption of neutral, public, open-source infrastruct
 
 The key words `MUST`, `MUST NOT`, `REQUIRED`, `SHALL`, `SHALL NOT`, `SHOULD`, `SHOULD NOT`, `RECOMMENDED`,  `MAY`, and `OPTIONAL` in this document are to be interpreted as described in [RFC-2119](https://www.ietf.org/rfc/rfc2119.txt).
 
+### Miner Configuration
+
+Miner should accept the following configuration options:
+* StrictProfitSwitch (int) - time in miliseconds to wait for both of the non-MEV (vanilla) and the `MEV block` construction before selecting the best available block for mining. If value is zero then no waiting is necessary.
+* ProxyContractAddress (address) - an address at which the proxy payment contract is deployed with the default value of 0xf1A54b075Fb71768ac31B33fd7c61ad8f9f7Dd18
+* MaxMergedBundles (int) - max number of `MEV bundles` to be included within a single block
+
 ### Definitions
 
-#### `Bundle`
-A set of transactions that `MUST` be executed together, `MUST` be executed before any non-bundle transactions only after other bundles that have a higher `simulated MEV equivalent gas price`, and `MUST` execute without failure (return status 1 on transaction receipts).
+#### `Relay`
+
+An external system delivering validated `MEV bundle` to the node.
```

Comment:
Removed validation (I meant scanning bundles for formatting issues). I removed this mention since we would prefer MEV Geth to have its own validation nonetheless.